### ISSUE TITLE
Permanent error

### ISSUE THREAD

**Issue by Glaived** [isaacKenyon/VALORANT-rank-yoinker#21]
v1.0.0

```
C:\Users\Glaived\AppData\Local\Programs\VALORANT rank yoinker>"VALORANT rank yoinker-Portable.exe"
Traceback (most recent call last):
  File "main.py", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "requests\__init__.py", line 100, in <module>
  File "requests\__init__.py", line 85, in check_compatibility
Exception: You need either charset_normalizer or chardet installed
[636] Failed to execute script 'main' due to unhandled exception!
```

**Comment by Glaived** [comment]
???

**Comment by OwOHamper** [comment]
Gonna be fixed in 1.2
